drivers/video/ov5640.c

```diff
@@ -980,6 +1210,13 @@ static int ov5640_init(const struct device *dev)
 
 	k_sleep(K_MSEC(20));
 
+	/* Reset all registers */
```

Comment:
What is this ? We need it for both modes ?

Reply:
This returns the registers to default values. Should I remove it?